src/platform/BUILD.gn

```
@@ -63,9 +63,18 @@ if (chip_device_platform != "none" && chip_device_platform != "external") {
     # Indication that the CHIP data model is included
     chip_enable_data_model = false
 
-    # The string of device software version was built.
+    # Device software version as an integer.  e.g. chip_device_config_device_software_version=7
+    chip_device_config_device_software_version = ""
+
+    # Device software version string as a string having the format "x.y"  e.g. chip_device_config_device_software_version="\"7.0\""
     chip_device_config_device_software_version_string = ""
 
+    # Device vendor id as an integer. Configurable vendor ids allowed are 65521 (0xFFF1), 65522 (0xFFF2), 655223 (0xFFF3).
```

Can it actually be the hex values?  Or does it need to be decimal?

---

i tried to use hex values but without success. it would just need more time to figure out so i let it be.

---

My point is that the documentation needs to say what values are actually allowed.  It does not clearly do that right now.
```suggestion
    # Device vendor id as a decimal integer. Configurable vendor ids allowed are 65521 (0xFFF1), 65522 (0xFFF2), 655223 (0xFFF3).
```